pkg/apis/submariner/v1alpha1/submariner_types.go

```diff
@@ -44,6 +44,8 @@ type SubmarinerSpec struct {
 	ServiceCIDR              string `json:"serviceCIDR"`
 	ClusterCIDR              string `json:"clusterCIDR"`
 	Namespace                string `json:"namespace"`
+	CableType                string `json:"cableType"`
```

Comment:
What do you think about using may be just CableType, which could be in the format:

cable[/driver]  ?

Reply:
I'd also add the omitempty, which makes it an optional field, for backwards compatibility I think we need it.

Reply:
Done